application/core/HtmlBuilder.php

```diff
@@ -1,655 +1,656 @@
+        }
+        if(!empty($class)){
+            print(" class=\"{$class}\"");
+        }
+        print(">");
+        if(!empty($options)){
+            $this->option($options, $selected);
+        }
+        print($close === true ? "</select>" : "");
+        
+        return $this;
+    }
+    
+    /**
+     * Builds the options for your select
+     * 
+     * @param   array, string
+     * @author  sbebbington
+     * @date    6 Jul 2017 - 12:12:56
+     * @version 0.1.4-RC3
+     * @return  $this
+     */
+    public function option(array $options, string $selected = ''){
+        foreach($options as $key => $data){
+            print("<option value=\"{$key}\"");
+            if($key == $selected){
+                print(" selected=\"selected\"");
+            }
+            print(">{$data}</option>");
+        }
+        
+        return $this;
+    }
+    
+    /**
+     * Used for opening any element not covered here
+     * i.e. in your view:
+     *         $this->open("div", "content", "col-xs-12", array('display' => "none"));
+     * will generate the following HTML:
+     *         <div id="content" class="col-xs-12" style="display: none;">
+     * The style attribute has been added for those
+     * who use [primarily] jQuery to make web pages
+     * dynamic
+     * 
+     * @param   string, string, string | array, string | array, [boolean]
+     * @author  sbebbington
+     * @date    30 Mar 2017 - 11:22:00
+     * @version 0.1.4-RC3
+     * @return  $this
+     */
+    public function open(string $element, string $id = '', $class = null, $style = null, bool $selfClose = false){
+        print("<{$element}");
+        if(strlen($id)>0){
+            $this->id($id);
+        }
+        if(is_string($class) || is_array($class)){
+            $this->addClass($class);
+        }
+        if(!empty($style) && (is_string($style) || is_array($style))){
+            $this->style($style);
+        }
+        $this->closeElement($selfClose);
+        
+        return $this;
+    }
+    
+    /**
+     * Used for closing an element such as a section
+     * or a div etc...
+     * i.e., in your view:
+     *         $this->close("div");
+     * will generate the following HTML:
+     *         </div>
+     * 
+     * @param   string
+     * @author  sbebbington
+     * @date    23 Jan 2017 - 10:07:49
+     * @version 0.1.4-RC3
+     * @return  $this
+     */
+    public function close(string $element){
+        print("</{$element}>");
+        
+        return $this;
+    }
+    
+    /**
+     * For HTML elements H1 to H6 inclusive
+     * the style attribute is added for use
+     * with jQuery and such
+     * 
+     * @param   int, string, string, string | array, string | array
+     * @author  sbebbington
+     * @date    30 Mar 2017 - 10:51:02
+     * @version 0.1.4-RC3
+     * @return  $this
+     */
+    public function h(int $size, string $text, string $id = '', $class = null, $style = null){
```

Comment:
Probably better not to use reserved words for function names i.e javascript, title, style etc